steps/move-or-copy-files/0.0.1/step.yml

```diff
@@ -0,0 +1,56 @@
+title: Move or Copy Files
+summary: |
+  Moves or copies files
+description: |
+  -
+website: https://github.com/...
```

Comment:
please fill these URLs too, can be simply the github page of the repo (`https://github.com/Dave005/bitrise-step-move-or-copy-files`)